### PR TITLE
189 i/o toggle

### PR DESCRIPTION
**Pull request by Nicvaniek**

### CODE REVIEW

File: src/main/python/image_preprocessing/pipeline.py [hunk 1]
```diff
@@ -17,45 +17,54 @@ def __init__(self, blur_manager=None, color_manager=None, face_detector=None, th
...
         Args:
+            useIO (boolean): Wheteher or not to write images to disk
             image (:obj:'OpenCV image'): Image to which processing should be applied to.
             remove_face :boolean: If the remove face flag is set to true extra processes will
                 be activated during the pre-processing phase to remove the face from the image.
         Returns:
             image: The processed image.
 
         """
+        print("-------------------------------------------------------")
```
Comment (AndreasNel) [inline src/main/python/image_preprocessing/pipeline.py]: No.

File: src/main/python/image_preprocessing/pipeline.py [hunk 2]
```diff
@@ -17,45 +17,54 @@ def __init__(self, blur_manager=None, color_manager=None, face_detector=None, th
         self.face_detector = face_detector
         self.threshold_manager = threshold_manager
 
-    def process_text_extraction(self, image, remove_face=False):
+    def process_text_extraction(self, useIO, image, remove_face=False):
```
Comment (AndreasNel) [inline src/main/python/image_preprocessing/pipeline.py:20]: Make `useIO` have a default value of `False`

**Review by AndreasNel — CHANGES_REQUESTED** [review]
2 small changes. One of which I am deeply disappointed in you 😉 and one that seems small, but will have a large impact on the rest of this pull request.

File: src/main/python/image_preprocessing/pipeline.py (hunk 1 above)
Comment (Nicvaniek) [reply src/main/python/image_preprocessing/pipeline.py]: 😂😂😂

Comment (nellstephanj) [reply src/main/python/image_preprocessing/pipeline.py]: 🤦🏼‍♂

File: src/main/python/image_preprocessing/pipeline.py [hunk 3]
```diff
@@ -10,52 +10,59 @@ class Pipeline:
...
         """
         This function applies all the processing needed to extract text from a image.
         Author(s):
             Nicolai van Niekerk
         Args:
+            useIO (boolean): Wheteher or not to write images to disk
```
Comment (AndreasNel) [inline src/main/python/image_preprocessing/pipeline.py]: Wheteher -> Whether

**Review by AndreasNel — CHANGES_REQUESTED** [review]

**Review by AndreasNel — APPROVED** [review]

**Pull request merged**